### PR TITLE
Be setup

### PR DESCRIPTION
**Pull request by kabcouwer**
## What we did:
- Updated doc/pull_request_template.md to DOCS/pull_request_template.md
- Added gems - shouldamatchers, factorybot, rspec-rails, faker, database_cleaner
- Added spec file structure

## Notes
- none

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -31,10 +31,18 @@ gem 'bootsnap', '>= 1.1.0', require: false
...
 group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
+end 
+
+group :test do
+  gem 'factory_bot_rails', '~> 4.0'
+  gem 'shoulda-matchers', '~> 3.1'
+  gem 'faker'
+  gem 'database_cleaner'
 end
```
Comment (marlitas) [inline Gemfile:46]: We should probably add webmock to this as well

**Review by marlitas — CHANGES_REQUESTED** [review]
Looks good, just add in webmock

**Comment by kabcouwer** [comment]
Lets add to the main:
gem 'faraday'
gem 'figaro'

development, test:
gem 'pry'
gem 'travis'

test:
gem 'simplecov'

**Review by marlitas — APPROVED** [review]
Looks good!

**Pull request merged**